docs/routing/special_routes.md

```diff
@@ -12,7 +12,7 @@ It is used to render errors, if an application with the `Primary` or `Essential`
   1. If there is no route configured in Registry for requested URL.
   2. The app fails to find the requested resource by it's ID (this rule is only for **primary** apps). More details about it [here](../global_errors_handling.md#404-error-not-found)
 
-- When using `Special route` all standard routes are ignored, including `*` with any `Order pos`. If you want something else to be displayed on the page besides the error (header, footer, etc.), you need to add them to the [Slot](./route_configuration_options.md#slot-configuration) as in the screenshot below (This is necessary because standard routes are ignored).
+-`Special route` ignores all standart routes, including `*` with any `Order pos`. This means that you must add all the necessary applications (header, footer, etc.) to the `Special route` [slot](./route_configuration_options.md#slot-configuration) (as in the screenshot below).
```

Comment:
As for me this part  "including `*` with any `Order pos`" a little bit confuses.
What do you think about replacing:
-`Special route` ignores all standart routes, including `*` with any `Order pos`.`

With this one:
`Unlike simple routes which use composition with the help of `next: true|false` property , the special route has own full configuration.`